flocker_bb/builders/flocker.py

```diff
@@ -128,10 +127,8 @@ def _flockerCoverage():
             ),
         DirectoryUpload(
             b"change-coverage-annotations",
-            Interpolate(b"private_html/%s/%s/change" % (branch, revision)),
-            url=Interpolate(
-                b"/results/%s/%s/change" % (
-                    branch, revision)),
+            resultPath('change-coverage'),
+            url=resultURL('change-coverage'),
```

Comment:
Oh cool. So is there a report showing only the coverage of the files changed in this branch / merge?
I found eg 
 * http://build.clusterhq.com/results/master/flocker-4006/change/

But not sure how to interpret the `cover` files in there.